django_project/certification/tests/model_factories.py

```diff
@@ -0,0 +1,166 @@
+# coding=utf-8
```

Comment:
hi @ann26 
it quite same
maybe you could create factories file based on factory instead put all of model test in one file.
for example certifying_organisation_factory.py just for class CertifyingOrganisationF